libgalois/include/katana/PropertyGraph.h

```diff
@@ -473,80 +487,200 @@ class KATANA_EXPORT PropertyGraph {
     return edge_properties()->schema();
   }
 
-  /// \returns the number of node types
-  size_t GetNodeTypesNum() const {
-    return node_type_name_to_type_set_ids_.size();
+  /// \returns the number of node atomic types
+  size_t GetNodeAtomicTypesNum() const {
```

Comment:
Since we are renaming: Having "Num" at the end reads weirdly. Could we have `GetNodeAtomicTypesCount` or `GetNumNodeAtomicTypes`?

Reply:
Same for several methods.

Reply:
That was the norm in this class. I don't mind changing it but it needs to be consistent. EIther `GetNum<x>` or `Num<x>` is fine with me.

Reply:
Agree with Arthur.